pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -528,9 +528,9 @@ func (r *reconciler) computeOFRulesForAdd(rule *CompletedRule, ofPriority *uint1
 			if r.fqdnController != nil && len(rule.To.FQDNs) > 0 {
 				var addresses []types.Address
 				addressSet := sets.NewString()
-				klog.InfoS("Ready to get IPs for FQDN Selector while adding a rule", "FQDN", rule.To.FQDNs)
-				matchedIPs := r.fqdnController.getIPsForFQDNSelectors(rule.To.FQDNs)
-				klog.InfoS("Got IPs for FQDN Selector while adding a rule", "FQDN", rule.To.FQDNs)
+				klog.Infof("Getting IPs for FQDN %v when updating rule %s", rule.To.FQDNs, rule.ID)
```

Comment:
Just curious why are we changing back to `Infof` instead of `InfoS`? To keep consistency with the rest of the file?

Reply:
@Dyanngg This is just test code right?